src/components/HTML/HTML.tsx

```diff
@@ -1,35 +1,61 @@
-import React from 'react';
+import React, {PropsWithChildren, useMemo} from 'react';
 
-import {QAProps} from '../../models/common';
-import {hasBlockTag} from '../../utils';
+import {ClassNameProps, QAProps} from '../../models/common';
+import {selectVariant} from '../../utils';
 
-export interface HTMLProps {
-    children?: string;
+export interface HTMLExtraProps {
+    variant?: 'span' | 'div' | 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6' | 'section' | 'p';
```

Comment:
and I think we need more informative name for this props , maybe `tagVariant` or `tagType`

Reply:
changed it to tagName